src/main/java/com/paneedah/weaponlib/EntityProjectile.java

```diff
@@ -230,7 +230,7 @@ public void onUpdate() {
                  IBlockState state = world.getBlockState(rtr.getBlockPos());
                  if(state.getMaterial() == Material.GLASS) {
                      if(ModernConfigManager.bulletBreakGlass)
-                         this.world.destroyBlock(rtr.getBlockPos(), true);
+                         this.world.destroyBlock(rtr.getBlockPos(), false);
```

Comment:
No, this change would break mods like Quark I think it's Quark, that adds glass shards, anyway vanilla default behavior with glass is that it drops nothing so this change would do nothing except break compat with mods.